### PR TITLE
Set links in main navigation

### PR DESCRIPTION
**Pull request by indigo423**
Add some useful links to resources in our main navigation bar in the docs framework.

### CODE REVIEW

**Comment by indigo423** [comment]
Commercial Services
Community Tools
Video Material

File: src/partials/header-content.hbs [hunk 1]
```diff
@@ -12,32 +12,35 @@
       <div class="navbar-end">
         <a class="navbar-item" href="#">Home</a>
         <div class="navbar-item has-dropdown is-hoverable">
-          <a class="navbar-link" href="#">Products</a>
+          <a class="navbar-link" href="#">Video Material</a>
```
Comment (Bonrob2) [inline src/partials/header-content.hbs]:
```suggestion
          <a class="navbar-link" href="#">Videos</a>
```

File: src/partials/header-content.hbs [hunk 2]
```diff
@@ -12,32 +12,35 @@
...
+            <a class="navbar-item" href="https://www.opennms.com" target="blank">Project Website</a>
+            <a class="navbar-item" href="https://issues.opennms.org" target="blank">JIRA</a>
+            <a class="navbar-item" href="https://opennms.discourse.group" target="blank">Discourse</a>
+            <a class="navbar-item" href="https://chat.opennms.com" target="blank">Mattermost Chat</a>
+            <a class="navbar-item" href="https://toolbox.opennms.eu" target="blank">Toolbox</a>
           </div>
         </div>
         <div class="navbar-item has-dropdown is-hoverable">
-          <a class="navbar-link" href="#">Resources</a>
+          <a class="navbar-link" href="#">Commercial Services</a>
```
Comment (Bonrob2) [inline src/partials/header-content.hbs]:
```suggestion
          <a class="navbar-link" href="#">OpenNMS Group</a>
```
This is repeated below, but I'm not sure what to specify instead. Suggest OpenNMS Group.

File: src/partials/header-content.hbs [hunk 3]
```diff
@@ -12,32 +12,35 @@
...
           <div class="navbar-dropdown">
-            <a class="navbar-item" href="#">Resource A</a>
-            <a class="navbar-item" href="#">Resource B</a>
-            <a class="navbar-item" href="#">Resource C</a>
+            <a class="navbar-item" href="https://www.opennms.com/services" target="blank">Commercial Services</a>
+            <a class="navbar-item" href="https://www.opennms.com/training" target="blank">Official Training</a>
```
Comment (Bonrob2) [inline src/partials/header-content.hbs]:
```suggestion
            <a class="navbar-item" href="https://www.opennms.com/training" target="blank">Training</a>
```

**Review by Bonrob2 — CHANGES_REQUESTED** [review]
Really nice.

**Pull request merged**